tests/jobserver/test_releases.py

```diff
@@ -20,163 +25,89 @@ def raise_error(**kwargs):
 
 
 @pytest.mark.django_db
-def test_handle_release_created():
+def test_create_release():
+    backend = BackendFactory(name="backend2")
     workspace = WorkspaceFactory()
-    backend = BackendFactory()
-    upload = ReleaseUploadFactory()
-
-    release, created = handle_release(
-        workspace,
-        backend,
-        "user",
-        upload.hash,
-        upload.zip,
-    )
-    assert created
-    assert release.id == upload.hash
-    assert release.backend_user == "user"
-    assert release.files.count() == 1
-    assert release.files.first().name == "file.txt"
-    assert (
-        release.files.first().path
-        == f"{workspace.name}/releases/{upload.hash}/file.txt"
-    )
-    assert release.manifest == {"workspace": "workspace", "repo": "repo"}
+    user = UserFactory()
+    releases.create_release(workspace, backend, user, {"file1.txt": "hash"})
```

Comment:
Should there be some asserts after this?

Reply:
do'h, although, there's not much to assert it was basically checking the db object was created without raising.